### DISCUSSION TITLE
Field defaults when adding an animal

### DISCUSSION THREAD

**Discussion by fopashelter** [sheltermanager/asm3#1452] [Q&A / Help]
We have defaults set up in the lookup lists for the Add an Animal screen.  We would like to remove the default to make the field display with no value by default.   In Settings-> OPtions -> Defaults, I am not seeing a way to clear a default and save it.  Anyone know if that is possible?

**Comment by Tetzor** [comment]
Hi,

There does have to be a default set. However you can add more options to the drop downs in Settings->Lookup Data. I wouldn't recommend creating a blank lookup and setting it as a default as it can cause confusion. Some people do create "Select One" or something similar to make it clear the person who created the record didn't select an option.
